Cargo.lock

```diff
@@ -423,6 +423,7 @@ dependencies = [
  "serde",
  "serde_derive",
  "serde_json",
+ "serde_repr",
```

Comment:
I don't think we are supposed to edit this file by hand, `cargo build` will take care of that

Reply:
That change was made by cargo build. But since I initially didn't push the updated `Cargo.lock` the Pipeline failed on `cargo build --locked`, since the Lockfile was out of date.